bee-network/bee-autopeering/src/local/mod.rs

```diff
@@ -0,0 +1,194 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+pub(crate) mod salt;
+pub mod services;
+
+use salt::Salt;
+use services::{ServiceMap, ServiceProtocol};
+
+use crate::peer::peer_id::PeerId;
+
+use crypto::signatures::ed25519::{PublicKey, SecretKey as PrivateKey, Signature, SECRET_KEY_LENGTH};
+use libp2p_core::identity::ed25519::Keypair;
+
+use std::{
+    fmt,
+    sync::{Arc, RwLock, RwLockReadGuard, RwLockWriteGuard},
+};
+
+use self::salt::SALT_LIFETIME_SECS;
```

Comment:
Why not group it with the other use salt:: ?

Reply:
good catch. Done